ImGuiRenderer.cpp

```diff
@@ -212,7 +223,7 @@ bool ImGuiRenderer::createDeviceObjects()
         "}\n";
 
     const GLchar* fragment_shader =
-        "#version 330\n"
+        IMGUIRENDERER_GLSL_VERSION
```

Comment:
I finally found the reason why no window is show in emulator.
`precision mediump float;` is needed in fragment shader.
(Seems `E eglCodecCommon: glUtilsParamSize: unknow param 0x000085b5` was not related)

Reply:
I added a line precision mediump float; in the fragment shader. Tell me if it is ok for you